content/user/apps/Flow-map.md

```diff
@@ -0,0 +1,46 @@
+<!--
+title: "Flow map"
+description: "Learn how to view the flow of data through your application."
+tags: "user UI applications flow map data access"
+-->
+
+The application flow map provides an interactive view of where data and resources are shared within your organization and beyond it.
+
+## How It Works
+
+Every time you exercise an application, the Contrast UI uses data reported from your Contrast agent to create a detailed map of how data flows from the application to other applications to which it's connected. The flow map organizes information for the application you're currently viewing into three connected sections: **Application Architecture**, **Back-End Systems** and **Connected Applications**. 
+
+By viewing all of the connections to your application, you can see the amount of data and resources that are associated with the application at full scale. By focusing on connections between individual systems and applications, you can also determine if users and connected applications in your organization have appropriate access to the current application and sensitive data associated with it. 
+
+<a href="assets/images/Flow-map-active.png" rel="lightbox" title="Flow Map tab"><img class="thumbnail" src="assets/images/Flow-map-active.png"/></a>
+
+The agent performs application matching through string credentials. If the agent isn't currently reporting data for the primary application, the **Back-End Systems** and **Connected Applications** sections are left blank.
+
+<!-- In what scenarios would the agent not report this info? Not exercised? -->
+
+<a href="assets/images/Flow-map-empty.png" rel="lightbox" title="No data in the Flow Map tab"><img class="thumbnail" src="assets/images/Flow-map-empty.png"/></a>
+
+## Application Architecture
+
+The **Application Architecture** section breaks down the view, presentation and service layers of the application's front end. You can also see foundational information about the application, including the environments in which it's deployed, letter grade, vulnerability statuses and attack status. 
+
+### Layers
+
+The **View** column displays the layer of technologies that determine what a browser sees and processes. The **Presentation** column displays the layer of technologies that generates the application view. The **Service** column displays the layer comprised of the database, LDAP driver or back end code performing the application logic.
+
+Hover on an item in any of the lists to see how many instances of each type of library are used in the application, or click on the library to go to the library's page in the UI. If the agent reports any vulnerabilities, a warning icon appears beside the library in which they were found; hover over the icon for links to the vulnerabilities' **Overview** pages.
+
```

Comment:
The mockups I saw referred to "libraries", but I know we also referred to these things as "technologies". Is one term better than the other?

Reply:
@kaitlynmortimer I think technologies is the broader term, but the fact that some of the columns are specifically libraries that we know far more information on and even link out to the detail pages we have for each within the UI, differentiates them quite a bit from the other columns that just show technologies, of which we know very little about beyond their name.